lightning-tag/detailed-reference.md

```diff
@@ -1,3 +1,111 @@
+
+The `data` object's parameters are fully described in [Lightning Tag Data Collection](lightning-tag/data-collection.md).
+
+## Retrieving Audiences
+
+The Lotame Lightning Tag allows you to retrieve audiences for the customer on your site. There are a few ways to do this and they are described below.
+
+### Callback
+
+As noted above in the `config` object created at script setup, there is a callback method named `onProfileReady` that can be set to be called at the completion of the targeting. This work happens asynchronously so the callback is fired when the targeting work is completed.
+
+When fired, the callback is passed a `profile` object that has the following functionality.
+
+Method Name | Return Type | Parameters | Description
+----------- | ----------- | ---------- | -----------
+`getProfileId()` | String | n/a | The id of the profile associated with the browser
+`getAudienceString( [delimiter=','], [limit])` | String | 1. `delimiter` - optional, string, defaults to comma. <br/> 2. `limit` - optional, integer, defaults to no limit | Delimited string of targeting codes or audience ids, based on your established tag settings. The default, i.e. no arguments, is comma delimited with no limit. Example: `"aud1,aud2"` or `"1,2"`
+`getAudiences([limit])` | Array | `limit` - optional, integer, defaults to no limit | Array of either targeting codes as strings or audience ids as integers, based on your established tag settings. Example: `["aud1","aud2"]` or `[123,789]`
+
+### Local Storage
+
+As noted above in the `config` object created at script setup, there is parameter named `audienceLocalStorage`. If passed, when targeting is completed, the audiences will be written out to the browser's localStorage.
+
+Value | Result
+:---: | ------
+`false` | This is the default and will not write audiences to browser localStorage.
+`true` |  Lightning Tag will write audiences to localStorage under the key `lotame_CLIENT_ID_auds` (where CLIENT_ID is your Lotame clientId)
+`'string'` | Passing a string will tell Lightning Tag to write the audiences to localStorage under the key of the string you pass. (i.e. `audienceLocalStorage: 'myLotameAudiences'`)
+
+Audiences are written out as a string of comma-delimited targeting codes or audience ids, based on your established tag settings. Example: `"aud1,aud2"` or `"123,789"`
+
+### getAudiences()
+
+?> Per Mark "which is a helper to retrieve audiences out of local storage". Need more details as I also see this method on the profile object returned by the callback.
+
+## Other Methods
+
+### Collect()
+
+The `collect()` method is available to pass data to your Lotame DMP after the page load is complete and the initial calls to Lotame DMP are finished. An example use-case is tracking user interaction with a video player on your site.
```

Comment:
An example use-case is tracking events such as user interaction with a video player on your site.

Reply:
Done.